source/community/release_notes/redwood.rst

```diff
@@ -13,9 +13,65 @@ These are the release notes for the Redwood release, the 18th community release
 Breaking Changes
 ****************
 
+-  Deployers must ensure that their
+   ``JWT_AUTH['JWT_PRIVATE_SIGNING_JWK']`` Django setting in LMS
+   contains the full complement of private key ``numbers.*``
 
-Learner Experiences
-*******************
+   -  Background: In LMS, we switched from the pyjwkest
+      library to PyJWT for signing JWTs. (pyjwkest is now unmaintained.)
+      However, PyJWT has stricter requirements for the private key in
+      ``JWT_PRIVATE_SIGNING_JWK``. Before you upgrade to Redwood, you
+      will need to update this key using a script. Otherwise, JWT
+      signing will fail, and users will be unable to log in.
+
+   -  Steps:
+
+      1. Locate ``JWT_PRIVATE_SIGNING_JWK`` in your deployment
+         configuration.
+
+      2. Check if the JSON contains all of the following params: ``p``,
+         ``q``, ``dp``, ``dq``, and ``qi``. If it does, you don’t need
+         to do anything further. Otherwise, continue.
+
+      3. In your edx-platform virtualenv, run
+         ``python3 scripts/jwk-precompute-params.py`` and follow the
+         prompts. (It will ask you to paste in the current JSON.)
+
+      4. Update your config’s ``JWT_AUTH['JWT_PRIVATE_SIGNING_JWK']``
+         with the output of the script.
+
+      5. You may wish to compare the contents of the private key before
+         and after running the script, and verify that the only changes
+         it has made to the contents of the JSON are that the ``p``,
+         ``q``, ``dp``, ``dq``, and ``qi`` params have been added. (Some
+         or all may already have been present.)
+
+   -  Notes:
+
+      1. This should be done while you are still running Quince—it is
+         safe to do ahead of the upgrade, and should not have any
+         visible effect at that time.
+
+      2. This key must be handled very carefully. Anyone in possession
+         of the key may impersonate users.
+
+- `studio_home.enable_global_staff_optimization flag no longer works in Studio MFE <https://github.com/openedx/wg-build-test-release/issues/380>`_
+   - Background: This flag works for the legacy Studio Home, improving performance by
+     adding an organization search bar.  This flag does not work in the Studio Home MFE, as it
+     is not needed within the MFE. The MFE already have a search bar and pagination, which
+     improves performance for those home pages with lots of courses.
+     adding an organization search bar. But as the report mentions, it's not working
+     correctly in the Studio Home MFE. The good thing is that this flag is not needed
+     in the Studio Home MFE since we already have a search bar and pagination, which
+     improves performance for those home pages with lots of courses. I wouldn't consider
+     it a "breaking change" but a known issue for those who want to turn on the Studio
+     Home MFE and are using the flag.
+
+Learner & Instructor Experiences
```

Comment:
TODO in this section:

We need to add additional notes about how to enable specific learner/instructor experiences.

* For example, I'd put the note about Meileisearch here, since it's related to enabling a specific experience.
* We need instructions on enabling in-course sidebar navigation. @crathbun428 can you provide info here?
* Make Sections available independently of the course outline - I'm waiting to hear more about this
* Connect Teams in a course to Content Groups - I'm waiting to hear more about this

Reply:
@mariajgrimaldi - Santiago says you can help with this one `Make Sections available independently of the course outline - I'm waiting to hear more about this`

Reply:
@sarina: I can help with `Connect Teams in a course to Content Groups`, but not with` Make Sections available independently of the course outline`, I'm not sure what that is.

I'll be on it in the next few hours.

Reply:
Alright. So for `Connect Teams in a course to Content Groups`:
1. Enable teams feature by turning on `teams.enable_teams_app`
2. Then, turn on the  `teams.content_groups_for_teams` waffle flag for everyone or for specific courses with a waffle flag course overrides


And for `Make Sections available independently of the course outline` enable this feature flag:
`FEATURES["ENABLE_HIDE_FROM_TOC_UI"] = True` to enable the feature system-wide.

Reply:
Thanks @mariajgrimaldi I added those as a suggestion below. Still need instructions about enabling the course sidebar (I know not your project, I'll get the info from Chelsea's team) but we're good on the other two!

Reply:
In case it helps, I found this waffle flag: `courseware.enable_navigation_sidebar` linked in this PR https://github.com/openedx/frontend-app-learning/pull/1398

Reply:
You're the best! I'll verify with Glib that this is all that needs to be done.